### PR TITLE
feat: adding and deleting notes function added

### PR DESCRIPTION
**Pull request by mdarif6**
feat: adding and deleting notes function added

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
Code Looks good, can be merged!

File: src/note-context.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { createContext, useContext } from "react";
+import { useReducer } from "react";
+
+const NoteContext = createContext();
```
Comment (PeeyushPrashant) [inline src/note-context.jsx:4]: Suggestion- you can create a single data-context for all functionalities like notes,acrhives, labels etc.
No need of creating separate context for each.

**Review by PeeyushPrashant — APPROVED** [review]
rest looks good
good folder structure

File: src/note-context.jsx (hunk 1 above)
Comment (mdarif6) [reply src/note-context.jsx:4]: thanks

**Pull request merged**